project1/index.html

```diff
@@ -0,0 +1,175 @@
+                            science"</em>
+                            during my time on
+                            the gulf. But as I waited to be picked up by my team in Corpus
+                            Christi, I
+                            was nervous—I
+                            imagined
+                            everyone
+                            else would
+                            be a scientist or a birder and have daunting binoculars.
+
+
+                            The biologist running the trip rolled
+                            up in in a large white van with a
+                            boat hitch
+                            and
+                            the words
+                            <a href=http://www.biologydiscussion.com/biological-sciences/biological-sciences-definition-history-and-objectives/85935>
+                                <i>BIOLOGICAL SCIENCES</i>
+            </a> stenciled
+            across the
+            side. Jeff was forty-ish, and wore sunglasses and a backward baseball cap. He had a winter
+            beard and a neon-green cast on his left arm. He’d broken his arm playing hockey with his
+            sons a week before. The first thing Jeff said was, “We’ll head back to camp,
+            but I hope
+            you
+            don’t mind we
+            run by the liquor
+            store
+            first.”
+            I felt more optimistic about my suitability for science.</p>
+
+
+
+
+
+            <p>Not long before I’d called off my engagement it was Christmas.
+
+                The woman who was supposed to be my mother-in-law was a wildly talented quilter and made
+                stockings with Beatrix
+                Potter characters on them for every family member. The previous Christmas she had
+                asked me what
+                character
+                I
+                wanted
+                to be (my fiancé was Benjamin Bunny). <i>I agonized over the decision. It felt important, like
+                    whichever character
+                    I
+                    chose would represent my role in this new family.</i> I chose <b><em>Squirrel Nutkin</em></b>, a
+                squirrel
+                with a
+                blazing red
+                tail—an
+                epic, adventuresome figure who ultimately loses his tail as the price for his daring and
+                pride.
+
+                I arrived in Ohio that Christmas and looked to the banister to see where my
+                squirrel had found his
+                place.
+                Instead, I
+                found <b><em>a mouse</em></b>. A mouse in a pink dress and apron. A mouse holding a broom and dustpan,
+                serious
+                about sweeping. A
+                mouse named Hunca Munca. The woman who was supposed to become my mother-in-law said, <blockquote>
+                    <em>“I was
```

Comment:
I would recommend against using the `strong` inside the `em` -- it's all a lot of markup and it starts to lose its meaning.

This is simpler, but more meaningful:
```
<blockquote>“I was going to do the  squirrel  but then I thought, that just isn’t CJ. <em>This is CJ.”</em></blockquote>
```